doc/release0.11.rst

```diff
@@ -1,4 +1,4 @@
-.. _release0.10:
+.. _release0.11:
```

Comment:
Hey after fixed this for some reason the link on 0.10 in the index.rst stopped linking.
I am now rerunning the website to double check.
Finally, we can compile the website and do other work simultaneously!! :8ball: